src/operationDefinitions/USCoreDocRef/index.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import express, { Router } from 'express';
+import { KeyValueMap, TypeOperation } from 'fhir-works-on-aws-interface';
+import { OperationDefinitionImplementation } from '../types';
+import ResourceHandler from '../../router/handlers/resourceHandler';
+import RouteHelper from '../../router/routes/routeHelper';
+import { DocRefParams, parsePostParams, parseQueryParams } from './parseParams';
+import { docRefParamsToSearchParams } from './docRefParamsToSearchParams';
+
+const searchTypeOperation: TypeOperation = 'search-type';
+
+const docRefImpl = async (resourceHandler: ResourceHandler, userIdentity: KeyValueMap, params: DocRefParams) => {
+    const searchParams = docRefParamsToSearchParams(params);
+    return resourceHandler.typeSearch('DocumentReference', searchParams, userIdentity);
```

Comment:
Should `userIdentity` be the fourth parameter that is passed to `typeSearch`?
https://github.com/awslabs/fhir-works-on-aws-routing/blob/b57b54c7ee0ef67799a14ffb6bc66a7e25977659/src/router/handlers/resourceHandler.ts#L62